src/main/java/teammates/ui/webapi/CreateFeedbackSessionLogAction.java

```diff
@@ -0,0 +1,23 @@
+package teammates.ui.webapi;
+
+/**
+ * Action: creates a feedback session log for the purposes of tracking and auditing.
+ */
+class CreateFeedbackSessionLogAction extends BasicFeedbackSubmissionAction {
+
+    @Override
+    AuthType getMinAuthLevel() {
+        return AuthType.LOGGED_IN;
```

Comment:
Why is it `AuthType.LOGGED_IN`? Shouldn't it be public because it should be called whenever a student views/submits a feedback session?

Reply:
Good point!